src/models/utils/flags.ts

```diff
@@ -25,6 +29,34 @@ import {
   TrustSetTransactionFlags,
 } from '../transactions/trustSet'
 
+/**
+ * Convert an AccountRoot Flags number into an interface for easy interpretation.
+ *
+ * @param flags - A number which is the bitwise and of all enabled AccountRootLedgerFlags.
+ * @returns An interface with all flags as booleans.
+ */
+// eslint-disable-next-line import/no-unused-modules -- Used by end user
+export function parseAccountRootFlags(
+  flags: number,
+): AccountRootLedgerFlagsInterface {
+  const flagsInterface = {
```

Comment:
Design question: Do we want to return every flag with either true/false, or do we want to only return flags set to true, with it implied that missing flags are false.

Reply:
Since they should be optional flags, I vote to only return flags set to `true`.